platform/android/MapboxGLAndroidSDK/build.gradle

```diff
@@ -157,6 +158,12 @@ def getRepositoryPassword() {
             (hasProperty('NEXUS_PASSWORD') ? NEXUS_PASSWORD : "")
 }
 
+def getGitRevision() {
```

Comment:
The other SDKs get this from a macro in mbgl. Probably not a big deal, but worth making sure the value is consistent at least.